test/lit/passes/outlining.wast

```diff
@@ -1006,3 +1006,57 @@
     (loop (nop))
   )
 )
+
+;; Test that no attempt is made to outline overlapping repeat substrings
```

Comment:
It would be good to add comments about what the overlapping substrings are.

Reply:
done